packages/server/src/registration/verifyRegistrationResponse.ts

```diff
@@ -288,6 +291,7 @@ export type VerifiedRegistrationResponse = {
     userVerified: boolean;
     credentialDeviceType: CredentialDeviceType;
     credentialBackedUp: boolean;
+    extensionsData?: AuthenticationExtensionsAuthenticatorOutputs;
```

Comment:
I hadn't paid all that much attention to extensions until now, so I'm trying to make sure your additions communicate things clearly to the library's users.

In this case I think it'd make sense to rename this to better communicate that these are **authenticator** extension outputs, instead of what comes in as `credential.clientExtensionResults` when `verifyRegistrationResponse()` is called:

```suggestion
    authenticatorExtensionResults?: AuthenticationExtensionsAuthenticatorOutputs;
```

Truth be told I'd rather this be called `authenticatorExtensionOutputs`, but there's already the concept of "Results" from `clientExtensionResults` here so we should probably go with that here too:

https://github.com/MasterKale/SimpleWebAuthn/blob/master/packages/typescript-types/src/index.ts#L71

Reply:
Oh, and same thing in `verifyAuthenticationResponse()` too